test/circlex/emulator/api/mocks_api_test.exs

```diff
@@ -13,11 +13,17 @@ defmodule Circlex.Emulator.Api.MocksApiTest do
             %{
```

Comment:
It's weird we return a raw map here, but I suppose it's because this is some strange endpoint that doesn't comply with a lot of other standards?

Reply:
yes. hard to test this endpoint because we don't know how VANs are generated, so using a raw map